test/Microsoft.Net.Http.Headers.Tests/NameValueHeaderValueTest.cs

```diff
@@ -575,6 +575,83 @@ public void TryParseStrictList_WithSomeInvlaidValues_ReturnsFalse()
             Assert.False(NameValueHeaderValue.TryParseStrictList(inputs, out results));
         }
 
+        [Theory]
+        [InlineData("value", "value")]
+        [InlineData("\"value\"", "value")]
+        [InlineData("\"hello\\\\\"", "hello\\")]
+        [InlineData("\"hello\\\"\"", "hello\"")]
+        [InlineData("\"hello\\\"foo\\\\bar\\\\baz\\\\\"", "hello\"foo\\bar\\baz\\")]
+        [InlineData("\"quoted value\"", "quoted value")]
+        [InlineData("\"quoted\\\"valuewithquote\"", "quoted\"valuewithquote")]
```

Comment:
Suggest verbatim strings to make the backslashes easier to see.

Reply:
It gets ugly with quotes (need to do ""), so I personally prefer these.